tests/actions.go

```diff
@@ -363,6 +365,36 @@ func (oi *OperatorConfig) OperatorHelmSetString(m map[string]string) string {
 	return strings.Join(arr, ",")
 }
 
+func (oa *operatorActions) runKubectlOrDie(args ...string) string {
+	cmd := "kubectl"
+	glog.Infof("Running '%s %s'", cmd, strings.Join(args, " "))
+	out, err := exec.Command(cmd, args...).CombinedOutput()
+	if err != nil {
+		glog.Fatalf("Failed to run '%s %s'\nCombined output: %q\nError: %v", cmd, strings.Join(args, " "), string(out), err)
+	}
+	glog.Infof("Combined output: %q", string(out))
+	return string(out)
+}
+
+func (oa *operatorActions) CleanCRDOrDie() {
+	oa.runKubectlOrDie("delete", "crds", "--all")
```

Comment:
for safety, just delete the crd we created.

Reply:
not necessary, what we require is a fresh cluster, in a fresh cluster, no CRDs is installed by default